cypress/support/commands.js

```diff
@@ -0,0 +1,42 @@
+// ***********************************************
```

Comment:
Do we want to keep all this commented out example stuff?

Reply:
We can remove it, but it's doesn't hurt anyone :)